web/source/text/prediction/modelManager.ts

```diff
@@ -92,6 +92,8 @@ namespace com.keyman.text.prediction {
     private recentTranscriptions: Transcription[] = [];
 
     private _enabled: boolean = true;
```

Comment:
Do we need a separate `_enabled` flag now that we have both `_enabledPredictions` and `_enabledCorrections`? I would prefer a function `enabled() { return _enabledPredictions || _enabledCorrections; }` to avoid data synchronisation issues.

Reply:
Still need to delete this line?